javascript/javascript3/week1/giphy.html

```diff
@@ -1,85 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset='utf-8'>
+  <head>
+    <meta charset="utf-8" />
     <title>Page Title</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1'>
-    
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
     <style>
-    #bodycontainer{
+      #bodycontainer {
         width: 966px;
         border: 2px solid darkgray;
         padding: 1%;
         font-family: roboto;
         font-size: 14px;
         line-height: 20px;
-    }
-    #map{
+      }
+      #map {
         margin-top: 10px;
         width: 1000px;
         height: 500px;
-    }
-    input[type="text"], input[type="number"]{
+      }
+      input[type="text"],
+      input[type="number"] {
         font-size: 14px;
         font-weight: 400;
         padding: 12px 8px;
         color: #2e2e2e;
-        background-color:#fff;
+        background-color: #fff;
         border: 1px solid#d2d6d9;
         border-radius: 5px;
         margin-right: 5px;
-    }
-    input[type="button"]{
+      }
+      input[type="button"] {
         padding: 13px 27px;
         font-size: 15px;
         color: #fff;
         background: #3ecf8e;
         border: none;
         border-radius: 5px;
         margin-right: 5px;
-    }
-    #alldata{
-    display:flex ;flex-wrap: wrap;overflow-y: scroll;
+      }
+      #alldata {
+        display: flex;
+        flex-wrap: wrap;
+        overflow-y: scroll;
 
-height: 256px;
-    }
-    #alldata img{margin: 5px;height: 130px;display: block;}
-#searcharea{padding-bottom: 0em !important;
-border-bottom: 0px none !important;
- }
-   
+        height: 256px;
+      }
+      #alldata img {
+        margin: 5px;
+        height: 130px;
+        display: block;
+      }
+      #searcharea {
+        padding-bottom: 0em !important;

```

Comment:
Check the CSS hierarchy, to minimize or avoid the usage of the !important. 
You are forcing the browser to render this style.  It should not be necessary.

Reply:
Yes, totally agree. I think i have used old boilerplate framework for css and html and topped it with new one. I modified it. Thankyou.